Reply/app/src/main/java/com/example/reply/ui/ReplyApp.kt

```diff
@@ -19,16 +19,21 @@ package com.example.reply.ui
 import androidx.compose.animation.AnimatedVisibility
 import androidx.compose.foundation.background
 import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Box
```

Comment:
This file is getting quite big. Why not split out some of the composables into separate files eg. ReplyNavGraph, ReplyInboxScreen, the navigation components

Reply:
Divided in parts to keep navigation components and actions in separate package.